client/my-sites/earn/home.tsx

```diff
@@ -261,6 +282,8 @@ export default connect< ConnectedProps, {}, {} >(
 			siteId: site.ID,
 			selectedSiteSlug,
 			isFreePlan: ! isCurrentPlanPaid( state, site.ID ),
+			isJetpack: isJetpackSite( state, site.ID ),
```

Comment:
Shall we make the prop `isJetpackNotAtomic` or do you think that we'll need these values separate for other cards?

Reply:
We _might_ need them later. It's not expensive to keep them separate for now.